File: cases/taint/python/model-summary-through-positive/flow.py
from bridge import hold, pass_through


def dfb_source():  # DFB-SOURCE: model-summary-through-input
    return "tainted"


def dfb_sink(value):  # DFB-SINK: model-summary-through-sink
    pass


def run():
    dfb_sink(pass_through(dfb_source()))


def unused():
    return (hold, pass_through)

File: cases/taint/javascript/model-summary-through-positive/Flow.js
function dfb_source() { // DFB-SOURCE: model-summary-through-input
  return "tainted";
}

function dfb_sink(value) {} // DFB-SINK: model-summary-through-sink

function run() {
  dfb_sink(Bridge.pass(dfb_source()));
}
